client/app/scripts/charts/edge-container.js

```diff
@@ -50,14 +56,9 @@ class EdgeContainer extends React.Component {
   }
 
   render() {
-    const { layoutPrecision, points } = this.props;
+    const { layoutPrecision } = this.props;
     const other = _.omit(this.props, 'points');
 
-    if (layoutPrecision === 0) {
```

Comment:
To clarify, we need the optimization with non-animated edges now, but may take it out in the future.